src/cli_plugin/install/pack.js

```diff
@@ -70,6 +69,10 @@ async function mergePackageData(settings, packages) {
     pkg.version = _.get(packageInfo, 'version');
     pkg.name = _.get(packageInfo, 'name');
     pkg.path =  resolve(settings.pluginDir, pkg.name);
+    pkg.kibanaVersion = _.get(packageInfo, 'kibana.version');
+    if (!pkg.kibanaVersion) {
+      pkg.kibanaVersion = pkg.version;
```

Comment:
I didn't make it required because in the past, we said that the version of the plugin must match the version of Kibana. I wanted to make sure that the plugins that followed that guideline weren't left in the cold. Making it optional allows for authors to implement the split between their version and the required version of kibana as it becomes necessary.